source/model/backtraceMetrics.ts

```diff
@@ -13,11 +13,13 @@ export class BacktraceMetrics {
   private uniqueEndpoint: string;
 
   private sessionId: string = uuid();
+  private readonly eventsErrorWarning =
+    'Could not submit metrics. Are stability metrics enabled on your instance? Enable metrics ingenstion or disable this feature by providing `{ enableMetricsSupport: false}` in options.';
```

Comment:
I think the solution that we provide in the message should focus on making sure metrics are enabled and that it's clear how to do so. Is there an option to enable this in the UI or only via code? Also, is there a way to check what enableMetricsSupport is set to? 

```suggestion
    'Unable to submit metrics. Make sure metrics are enabled for your project with `{ enableMetricsSupport}` in options.';
```

Does "options" refer to the BacktraceClient options? If so we should specify that.